integration-cli/docker_cli_exec_test.go

```diff
@@ -240,44 +176,20 @@ func (s *DockerSuite) TestExecTtyCloseStdin(c *check.C) {
 		c.Fatal(out, err)
 	}
 
-	cmd = exec.Command(dockerBinary, "top", "exec_tty_stdin")
-	out, _, err := runCommandWithOutput(cmd)
-	if err != nil {
-		c.Fatal(out, err)
-	}
-
+	out, _ := dockerCmd(c, "top", "exec_tty_stdin")
 	outArr := strings.Split(out, "\n")
 	if len(outArr) > 3 || strings.Contains(out, "nsenter-exec") {
-		// This is the really bad part
-		if out, _, err := runCommandWithOutput(exec.Command(dockerBinary, "rm", "-f", "exec_tty_stdin")); err != nil {
-			c.Fatal(out, err)
-		}
-
 		c.Fatalf("exec process left running\n\t %s", out)
 	}
-
 }
 
 func (s *DockerSuite) TestExecTtyWithoutStdin(c *check.C) {
-
-	cmd := exec.Command(dockerBinary, "run", "-d", "-ti", "busybox")
-	out, _, err := runCommandWithOutput(cmd)
-	if err != nil {
-		c.Fatalf("failed to start container: %v (%v)", out, err)
-	}
-
+	out, _ := dockerCmd(c, "run", "-d", "-ti", "busybox")
 	id := strings.TrimSpace(out)
 	if err := waitRun(id); err != nil {
 		c.Fatal(err)
 	}
 
-	defer func() {
-		cmd := exec.Command(dockerBinary, "kill", id)
```

Comment:
I think this defer function is for container cleanup which is not needed now, so I removed it. Maybe I shouldn't change any logics in this PR?